src/modules/core/components/Button/Button.css

```diff
@@ -322,3 +322,19 @@
 .themeDottedArea:hover {
   opacity: 0.7;
 }
+
+/*
+ * White
+ * buttons with white background, --temp-grey-blue-7 border & font color
+ */
+
+.themeWhite {
```

Comment:
Are you sure `themeGhost` doesn't do exactly this ?

Reply:
Yeah, I am sure. You can actually check if you change 'white' for 'ghost' in the file. The differences are:
- transparent background vs. white background
- font, border color
- hover background